src/main/java/com/educare/unitylend/dao/StatusRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.educare.unitylend.dao;
+
+import com.educare.unitylend.model.Status;
+import org.apache.ibatis.annotations.Mapper;
+import org.apache.ibatis.annotations.Param;
+import org.apache.ibatis.annotations.Select;
+import org.springframework.stereotype.Repository;
+
+@Mapper
+@Repository
+public interface StatusRepository {
```

Comment:
Is there any need for the status Repository, Is it getting utilized anywhere?
Ideally it should not get used separately.

While fetching from any table which utilizes status table, You should ideally fetch the string status and nothing else (i.e - status code), and make validations in service layer based on status string.